.evergreen/debian_package_build.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+#
+# Test mongocxx's Debian packaging scripts.
+#
+# Supported/used environment variables:
+#   IS_PATCH    If "true", this is an Evergreen patch build.
+
+set -o xtrace
+set -o errexit
+
+on_exit () {
+  if [ -e ./unstable-chroot/debootstrap/debootstrap.log ]; then
+    echo "Dumping debootstrap.log"
+    cat ./unstable-chroot/debootstrap/debootstrap.log
+  fi
+}
+trap on_exit EXIT
+
+if [ "${IS_PATCH}" = "true" ]; then
+  git diff HEAD -- . ':!debian' > ../upstream.patch
+  git diff HEAD -- debian > ../debian.patch
+  git clean -fdx
+  git reset --hard HEAD
+  if [ -s ../upstream.patch ]; then
+    [ -d debian/patches ] || mkdir debian/patches
+    mv ../upstream.patch debian/patches/
+    echo upstream.patch >> debian/patches/series
+    git add debian/patches/*
+    git commit -m 'Evergreen patch build - upstream changes'
+    git log -n1 -p
+  fi
+  if [ -s ../debian.patch ]; then
+    git apply --index ../debian.patch
+    git commit -m 'Evergreen patch build - Debian packaging changes'
+    git log -n1 -p
+  fi
+fi
+
+cd ..
+
+git clone https://salsa.debian.org/installer-team/debootstrap.git debootstrap.git
+export DEBOOTSTRAP_DIR=`pwd`/debootstrap.git
+sudo -E ./debootstrap.git/debootstrap unstable ./unstable-chroot/ http://cdn-aws.deb.debian.org/debian
+cp -a mongo-cxx-driver ./unstable-chroot/tmp/
+sudo chroot ./unstable-chroot /bin/bash -c "(set -o xtrace && \
+  apt-get install -y build-essential git-buildpackage fakeroot debhelper cmake curl ca-certificates libboost-dev libsasl2-dev libicu-dev doxygen && \
+  mkdir /tmp/mongo-c-driver && \
+  curl -o deb.tar.gz -L https://s3.amazonaws.com/mciuploads/mongo-c-driver/master/mongo-c-driver-debian-packages-1.17.0-pre-mongo_c_driver_44906882c3f2bbd2a3605e359814e74e8fe40ec9.tar.gz && \
```

Comment:
The C driver's debian-package-build task uploads to a fixed latest URL. For example [this task](https://evergreen.mongodb.com/task/mongo_c_driver_packaging_debian_package_build_49d5590b0164e629c6adc1ed1b88e6852f62bba9_20_07_07_16_23_59) has this attachment:
https://s3.amazonaws.com/mciuploads/mongo-c-driver/master/mongo-c-driver-debian-packages-latest.tar.gz

Reply:
Thanks for the pointer! Done.